web/viewer.css

```diff
@@ -1850,7 +1817,14 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   }
 }
 
-@media all and (max-width: 860px) {
+@media all and (max-width: 900px) {
+  #toolbarViewerMiddle {
+    display: table;
+    margin: auto;
+    left: auto;
+    position: inherit;
+    transform: none;
+  }
   .sidebarOpen .hiddenMediumView {
```

Comment:
This is obviously a long standing issue, thus not really relevant to the current PR, but I'm just now realizing that these `.sidebarOpen .{hidden, visible}{Large, Medium}View` rules probably should be amended with the corresponding rules for `.sidebarMoving` as well.
When testing the patch with the sidebar `transition-duration: 200ms;` increased ten times (to be able to check that nothing strange happens during animation), I noticed that when the sidebar is collapsed some elements appear too soon because of the missing `.sidebarMoving` rules. In practice this isn't that noticeable given the short transition duration, but we might want to address it nonetheless.

Obviously this is an *existing* issue, so if you don't want to address that in this PR that's totally fine (otherwise a separate commit with those rules added might work).